restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/util/RCUtils.java

```diff
@@ -40,40 +43,47 @@
 public class RCUtils {
    private static final String TAG = "RCUtils";
 
-   public static void validateDeviceParms(HashMap<String, Object> parameters) throws RCException
+   public static void validateDeviceParms(HashMap<String, Object> parameters, Map<String, ?> storageManagerInterfaceParams) throws RCException
```

Comment:
Can we simplify that? Right now it's difficult follow the logic of validations. And I'm concerned it will be easy to break when adding more stuff